### ISSUE TITLE
[SOLVED] Error: `No Chrome installations found`

### ISSUE THREAD

**Issue by elchiconube** [oblador/loki#178]
```
loki test v0.18.1
  ❯ Chrome (app)
    ✖ Start
      → No Chrome installations found.
      Fetch list of stories
      Test chrome.laptop
      Stop
No Chrome installations found.
```
Not sure why but today I found this problem, maybe it's because I've updated my MacOS to Catalina?

Thanks

**Comment by elchiconube** [comment]
**Solved**

I've updated the package.json file more concrete  `"chrome-launcher": "^0.11.2"`  and it works fine, please, update it

**Comment by oleggromov** [comment]
Worked for me as well, so I am upvoting this.

**Comment by oblador** [comment]
Hey, 

It [is upgraded](https://github.com/oblador/loki/blob/master/packages/target-chrome-app/package.json#L24) just need to make a new release. Next version is a big refactor into a multipackage monorepo and a new UI, so I want to get it right so I don't break stuff. Thanks for your patience!

**Comment by susannakosic** [comment]
Hi, I am running into this issue as well.

**System info**
Catalina 10.15.3
node v 10.18.0
storybook/react 5.2.1
loki 0.17.0
chrome-launcher 0.12.0
Chrome 82.0.4061.2 (Official Build) canary (64-bit)

** Issue **
When running `loki test` I get error below:
`No Chrome installations found.
Error
    at new LauncherError (/Users/susanna.kosic/Dev/marketplace/gms_frontend/node_modules/loki/node_modules/chrome-launcher/dist/utils.js:35:22)
    at new ChromeNotInstalledError (/Users/susanna.kosic/Dev/marketplace/gms_frontend/node_modules/loki/node_modules/chrome-launcher/dist/utils.js:66:9)
    at Launcher.<anonymous> (/Users/susanna.kosic/Dev/marketplace/gms_frontend/node_modules/loki/node_modules/chrome-launcher/dist/chrome-launcher.js:145:27)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/susanna.kosic/Dev/marketplace/gms_frontend/node_modules/loki/node_modules/chrome-launcher/dist/chrome-launcher.js:9:58)`

**Comment by susannakosic** [comment]
> **Solved**
> 
> I've updated the package.json file more concrete `"chrome-launcher": "^0.11.2"` and it works fine, please, update it

This solution does not work for me. See system details in [below comment](https://github.com/oblador/loki/issues/178#issuecomment-587422452)

**Comment by oblador** [comment]
Fix is out in [0.19.0](https://github.com/oblador/loki/releases/tag/v0.19.0)
